robotools/evotools/__init__.py

```diff
@@ -226,12 +226,16 @@ def _prepare_evo_aspirate_dispense_parameters(
     tips : list of int
         Tip(s) that will be selected (out of tips 1-8)
     """
-    # required parameters
     if labware is None:
         raise ValueError("Missing required paramter: labware")
     if not isinstance(labware, liquidhandling.Labware):
         raise ValueError(f"Invalid labware: {labware}")
 
+    if wells is None:
+        raise ValueError("Missing required paramter: wells")
+    if not isinstance(wells, typing.Union[str, typing.Sequence[str], numpy.ndarray]):
```

Comment:
I tried something similar a few days ago but it didn't work: Things like `Sequence[str]` can't be used in `isinstance`

```suggestion
    if not isinstance(wells, (str, list, tuple, numpy.ndarray)):
```

Reply:
Also, there's no need to be too detailed about the argument checking because it adds a lot of boilerplate code. Things like `is not None` can easily be caught by `mypy`..

Reply:
I removed the `None` test from tests.py